test/file.e2e-spec.ts

```diff
@@ -1,25 +1,75 @@
-import { Connection } from 'cypher-query-builder';
 import * as request from 'supertest';
-import { FileNodeType } from '../src/components/file/dto';
-import { User } from '../src/components/user';
+
+import { File, FileNodeType } from '../src/components/file/dto';
 import {
   TestApp,
-  createTestApp,
   createSession,
+  createTestApp,
+  createUser,
   fragments,
 } from './utility';
-import { gql } from 'apollo-server-core';
 import { generate, isValid } from 'shortid';
 
+import { Connection } from 'cypher-query-builder';
+import { User } from '../src/components/user';
+import { createFile } from './utility/create-file';
+import { gql } from 'apollo-server-core';
+
 describe('File e2e', () => {
   let app: TestApp;
 
   beforeAll(async () => {
     app = await createTestApp();
   });
 
-  it('read file node by id', async () => {
+  afterAll(async () => {
+    await app.close();
+  });
+
+  it('Create FileNode', async () => {
     const token = await createSession(app);
```

Comment:
I re ordered these so Create happens before Read.  token is not used by this function